images/scripts/lib/build-deps.sh

```diff
@@ -32,8 +32,11 @@ EXTRA_DEPS="libappstream-glib rpmlint"
 
 # nodejs for starter-kit and other projects which rebuild webpack during RPM build; but it can't be installed on RHEL 8
 # mocks due to missing modularity package indexes
+# libappstream-glib-devel is needed for merging translations in AppStream XML files in starter-kit and derivatives
+# on RHEL 8 only: gettext in RHEL 8 does not know about .metainfo.xml files, and libappstream-glib-devel
+# provides /usr/share/gettext/its/appdata.{its,loc} for them
 case "$OS_VER" in
-    rhel*8) ;;
+    rhel*8) EXTRA_DEPS="$EXTRA_DEPS libappstream-glib-devel" ;;
```

Comment:
We need this on `centos*8` as well, but there we also need `nodejs`. So this should better become its own separate `case`/comment block.